scripts/xcpng-fs-diff.py

```diff
@@ -228,9 +244,14 @@ def compare_data(ref, test, show_diff):
                 continue
 
             if ref_data[dtype][file] != test_data[dtype][file]:
-                print("{} differs: {}".format(dtype, file))
-                if show_diff:
-                    remote_diff(ref_host, test_host, file)
+                print("{} differs: {}".format(dtype, file), end='')
+                if dtype == 'package':
+                    print(" (ref={}, test={})".format(ref_data[dtype][file],
+                                                      test_data[dtype][file]))
+                else:
+                    print("")
+                    if show_diff:
+                        remote_diff(ref_host, test_host, file)
```

Comment:
If show_diff is false, you just print a newline? I haven't checked the rest of the script so maybe it makes sense.